### PR TITLE
PERK-971 -- add disputes endpoints to go-sdk

### PR DESCRIPTION
**Pull request by joshraphael**

### CODE REVIEW

File: pkg/moov/disputes.go [hunk 1]
```diff
@@ -135,3 +170,47 @@ func (c Client) GetDispute(ctx context.Context, disputeID string) (*Dispute, err
 
 	return CompletedObjectOrError[Dispute](resp)
 }
+
+// AcceptDispute Accepts a dispute.
```
Comment (adamdecaf) [inline pkg/moov/disputes.go:174]: Nit: Is there more to explain here? This reads like you're using the word to define the word.

File: pkg/moov/disputes_test.go [hunk 2]
```diff
@@ -61,3 +61,69 @@ func Test_GetDisputes_NotFound(t *testing.T) {
 
 	require.Equal(t, moov.StatusNotFound, httpErr.Status())
 }
+
+func Test_AcceptDisputes_Unauthorized(t *testing.T) {
+	mc := NewTestClient(t)
+
+	// We don't have any disputes to test against! So using a random id to accept will return unauthorized since we dont own the resource
```
Comment (adamdecaf) [inline pkg/moov/disputes_test.go:68]: Can't we create a dispute in tests?

Comment (joshraphael) [reply pkg/moov/disputes_test.go:68]: Theres no endpoint to create a dispute on our docs https://docs.moov.io/api/money-movement/disputes/

and there are none existing in staging that i can test agains on myaccount. This comment was copied from the existing tests for disputes

File: pkg/moov/disputes.go (hunk 1 above)
Comment (joshraphael) [reply pkg/moov/disputes.go]: Sure, i was taking from the official text on the webpage but i added a little more

**Review by adamdecaf — APPROVED** [review]

**Pull request merged**